pkg/etc/init.d/wavefront-proxy

```diff
@@ -102,7 +102,14 @@ jsvc=$proxy_dir/bin/jsvc
 
 jsvc_exec()
 {
+    if [[ ! $1 == "-stop" ]]; then
+        > $daemon_log_file
```

Comment:
don't you want to append to this file if it already exists, instead of overwriting it each time?